### DISCUSSION TITLE
Keybinders and fullscreen mode on the terminal...

### DISCUSSION THREAD

**Discussion by quantum-quirks** [smacke/subsync#157] [Ideas]
I was thinking of improving this software by introducing keybinders and rendering the app in full_screen on the terminal using [Quo](https://pypi.org/project/quo)

**Comment by smacke** [comment]
It's a cool direction. I might suggest taking a look at https://github.com/Textualize/textual if you're interested in taking it on. I'd be supportive of PRs, provided they don't cause any regressions in sync quality, code quality, or compatibility.
